### PR TITLE
Add a README for the content directory

### PR DESCRIPTION
**Pull request by jchristgit**

### CODE REVIEW

File: pydis_site/apps/content/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# The "content" app
+
+This application serves static, markdown-based content. Django-wise there is
+relatively little code in there, most of it is concerned with serving our
```
Comment (MarkKoz) [inline pydis_site/apps/content/README.md]: This is a comma splice.
```suggestion
relatively little code in there; most of it is concerned with serving our
```

File: pydis_site/apps/content/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# The "content" app
+
+This application serves static, markdown-based content. Django-wise there is
```
Comment (MarkKoz) [inline pydis_site/apps/content/README.md]: All references to "Markdown" should be capitalised.

File: pydis_site/apps/content/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+relatively little code in there, most of it is concerned with serving our
+content.
+
+The markdown files hosting our content can be found in the
+[`resources/`](./resources) directory, and contain
```
Comment (MarkKoz) [inline pydis_site/apps/content/README.md]: Incomplete sentence?

File: pydis_site/apps/content/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+
+## Contributing pages
+
+Contributing pages is covered extensively in our online guide, which you can
+find
```
Comment (MarkKoz) [inline pydis_site/apps/content/README.md]:
```suggestion
The process of contributing to pages is covered extensively in our online guide, which you can
find
```

File: pydis_site/apps/content/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+[here](https://www.pythondiscord.com/pages/guides/pydis-guides/how-to-contribute-a-page/)
+or alternatively read it directly at
```
Comment (MarkKoz) [inline pydis_site/apps/content/README.md]: Split the sentence up so it doesn't get so long.
```suggestion
Alternatively, read it directly at
```

File: pydis_site/apps/content/README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+- `resources` contains the static Markdown files that make up our site's
+  [pages](https://www.pythondiscord.com/pages/)
+
+- `migrations` contains standard Django migrations. As the `content` app
+  contains purely static markdown files, no migrations are present here.
+
+- `tests` contains unit tests for verifying that the app works properly.
+
+- `views` contains our Django views generating and serving the pages from the
```
Comment (MarkKoz) [inline pydis_site/apps/content/README.md]:
```suggestion
- `views` contains Django views which generate and serve the pages from the
```

File: pydis_site/apps/content/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+  input markdown.
+
+As for the modules, apart from the standard Django modules in here, the
+`utils.py` module contains utility functions for discovering markdown files to
+serve on the filesystem.
```
Comment (MarkKoz) [inline pydis_site/apps/content/README.md]: I don't think this clarification is necessary, and it introduces ambiguity: are files being served on the filesystem, or are files, which are on the filesystem, being served? Of course the former doesn't even make sense, but it's confusing regardless.

```suggestion
serve.
```

File: pydis_site/apps/content/README.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+## Directory structure
+
+Let's look at the structure in here:
+
+- `resources` contains the static Markdown files that make up our site's
+  [pages](https://www.pythondiscord.com/pages/)
+
+- `migrations` contains standard Django migrations. As the `content` app
+  contains purely static markdown files, no migrations are present here.
```
Comment (MarkKoz) [inline pydis_site/apps/content/README.md]: Why do we even keep this directory around then? Will Django complain if it can't find it?

**Review by MarkKoz — CHANGES_REQUESTED** [review]

File: pydis_site/apps/content/README.md (hunk 8 above)
Comment (jchristgit) [reply pydis_site/apps/content/README.md]: Good point. I've removed it in 4ad92b98a0df047b12e302ce8d849c2bd85862f1.

File: pydis_site/apps/content/README.md (hunk 7 above)
Comment (jchristgit) [reply pydis_site/apps/content/README.md]: ca2c583b15d46d7821a03ec047678429f96c7e03

File: pydis_site/apps/content/README.md (hunk 6 above)
Comment (jchristgit) [reply pydis_site/apps/content/README.md]: ca2c583b15d46d7821a03ec047678429f96c7e03

File: pydis_site/apps/content/README.md (hunk 5 above)
Comment (jchristgit) [reply pydis_site/apps/content/README.md]: ca2c583b15d46d7821a03ec047678429f96c7e03

File: pydis_site/apps/content/README.md (hunk 4 above)
Comment (jchristgit) [reply pydis_site/apps/content/README.md]: ca2c583b15d46d7821a03ec047678429f96c7e03

File: pydis_site/apps/content/README.md (hunk 3 above)
Comment (jchristgit) [reply pydis_site/apps/content/README.md]: ca2c583b15d46d7821a03ec047678429f96c7e03

File: pydis_site/apps/content/README.md (hunk 2 above)
Comment (jchristgit) [reply pydis_site/apps/content/README.md]: ca2c583b15d46d7821a03ec047678429f96c7e03

File: pydis_site/apps/content/README.md (hunk 1 above)
Comment (jchristgit) [reply pydis_site/apps/content/README.md]: ca2c583b15d46d7821a03ec047678429f96c7e03

File: pydis_site/apps/content/README.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+## Directory structure
+
+Let's look at the structure in here:
+
+- `resources` contains the static Markdown files that make up our site's
+  [pages](https://www.pythondiscord.com/pages/)
+
+- `tests` contains unit tests for verifying that the app works properly.
+
+- `views` contains Django views which generating and serve the pages from the
```
Comment (MarkKoz) [inline pydis_site/apps/content/README.md]:
```suggestion
- `views` contains Django views which generate and serve the pages from the
```

**Review by MarkKoz — CHANGES_REQUESTED** [review]
Just one thing you forgot. Looks good otherwise.

File: pydis_site/apps/content/README.md (hunk 9 above)
Comment (jchristgit) [reply pydis_site/apps/content/README.md]: Sorry Mark, I have the brain of a parrot. Rest assured this mistake is fixed and won't happen again in the future.

Thanks for the review!

**Review by MarkKoz — APPROVED** [review]

**Review by kosayoda — APPROVED** [review]
rocks

**Pull request merged**